src/data/markdown/translated-guides/en/02 Using k6/03 Checks.md

```diff
@@ -3,16 +3,23 @@ title: 'Checks'
 excerpt: 'Checks are like asserts but differ in that they do not halt the execution, instead, they just store the result of the check, pass or fail, and let the script execution continue.'
 ---
 
-[*Checks*](/javascript-api/k6/check/) are like assertions, but they don't halt execution.
```

Comment:
Having worked quite a bit around checks and thresholds in the last months, I ALWAYS have to check which is halting the execution, and which is not.

From my perspective, this information is major, and should remain in the first sentence, maybe with a formulation that mirrors the one you rephrased about thresholds? 🙇🏻